js/handlers.js

```diff
@@ -20,12 +20,27 @@ weechat.factory('handlers', ['$rootScope', '$log', 'models', 'plugins', 'notific
     };
 
     // inject a fake buffer line for date change
-    var injectDateChangeMessage = function(message, buffer, date) {
-        var content = "Date changed to " + date.toDateString();
+    var injectDateChangeMessage = function(message, buffer, d1, d2) {
```

Comment:
`d1` and `d2` aren't particularly helpful variable names. We try to keep them as meaningful as possible, would you mind changing them to `prev_date` and `curr_date` or `old_date` and `new_date` or something along those lines? And maybe `next_date` or `day_after_new`/`day_after_curr` for `d1_plus_one`?